packages/Liquid-Tests.package/LQPollsDraftBuilderTests.class/instance/testMultipleDuplicates.st

```diff
@@ -0,0 +1,8 @@
+test
+testMultipleDuplicates
+
+	| original onlyDuplicates |
+	
+	original := {1 . 1 . 2 . 3 . 4 . 5 . 5 . 5 . 5 . 10} asOrderedCollection.
```

Comment:
Please use strings instead of numbers as the actual choices will be strings. Of course you can test additionally, if that works with numbers as well.